### PR TITLE
Renamed pages to routes

### PR DESCRIPTION
**Pull request by ReginaVarybrus**

### CODE REVIEW

File: src/routes/ChatRoute/ChatRoute.jsx [hunk 1]
```diff
@@ -8,9 +8,9 @@ export default function ChatPage() {
   return (
     <Container>
       <SearchBar />
-      <ChatWrapper>
+      <ChatStyled>
```
Comment (DimaShubin) [inline src/routes/ChatRoute/ChatRoute.jsx]: ChatStyled should be moved inside Chat component.

**Review by DimaShubin — CHANGES_REQUESTED** [review]

File: src/routes/ChatRoute/ChatRoute.jsx (hunk 1 above)
Comment (ReginaVarybrus) [reply src/routes/ChatRoute/ChatRoute.jsx]: Changed.
And in the next commit I also changed the functions in the routes to arrow functions.

File: src/redux-store/slices/dist/authSlice.dev.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+"use strict";
```
Comment (DimaShubin) [inline src/redux-store/slices/dist/authSlice.dev.js]: Delete this file.

File: src/routes/AccountRoute/AccountRoute.jsx [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import Profile from '@/components/Profile/Profile';
 
-export default function AccountPage() {
+const AccountRoute = () => {
   return (
     <div>
```
Comment (DimaShubin) [inline src/routes/AccountRoute/AccountRoute.jsx]: Redundant div

File: src/routes/LoginRoute/LoginRoute.jsx [hunk 4]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import LoginForm from '@/components/LoginForm/LoginForm';
 
-export default function LoginPage() {
+const LoginRoute = () => {
   return (
     <div>
```
Comment (DimaShubin) [inline src/routes/LoginRoute/LoginRoute.jsx]: Redundant div

File: src/routes/RegisterRoute/RegisterRoute.jsx [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import RegisterForm from '@/components/RegisterForm/RegisterForm';
 
-export default function RegisterPage() {
+const RegisterRoute = () => {
   return (
     <div>
```
Comment (DimaShubin) [inline src/routes/RegisterRoute/RegisterRoute.jsx]: Redundant div

**Review by DimaShubin — CHANGES_REQUESTED** [review]

File: src/redux-store/slices/dist/authSlice.dev.js (hunk 2 above)
Comment (ReginaVarybrus) [reply src/redux-store/slices/dist/authSlice.dev.js]: Done

File: src/routes/AccountRoute/AccountRoute.jsx (hunk 3 above)
Comment (ReginaVarybrus) [reply src/routes/AccountRoute/AccountRoute.jsx]: Done

File: src/routes/LoginRoute/LoginRoute.jsx (hunk 4 above)
Comment (ReginaVarybrus) [reply src/routes/LoginRoute/LoginRoute.jsx]: Done

File: src/routes/RegisterRoute/RegisterRoute.jsx (hunk 5 above)
Comment (ReginaVarybrus) [reply src/routes/RegisterRoute/RegisterRoute.jsx]: Done

File: src/routes/LoginRoute/LoginRoute.jsx [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import LoginForm from '@/components/LoginForm/LoginForm';
+
+const LoginRoute = () => {
+  return <LoginForm />;
```
Comment (DimaShubin) [inline src/routes/LoginRoute/LoginRoute.jsx:5]: Probably files like that one don't have any sense but let's leave it as is for now.

**Review by DimaShubin — APPROVED** [review]

**Pull request merged**